src/praisonai/praisonai/cli/main.py

```
@@ -5296,6 +5299,27 @@ def create_realtime_interface(self):
+        Create an aiui-based agents interface (replaces Chainlit).
+        
+        Routes to the new `praisonai ui agents` subcommand.
+        """
+        try:
+            from praisonai.cli.commands.ui import _launch_aiui_app
+            print("🤖 Launching PraisonAI Agents Dashboard (aiui)...")
+            _launch_aiui_app(
+                app_dir="ui_agents",
+                default_app_name="ui_agents",
+                port=8082,  # Use same port as old Chainlit agents
+                host="0.0.0.0",
```

Ruff (0.15.10)

[error] Possible binding to all interfaces

(S104)